test/openshift/e2e/parallel/1-019_validate_volume_mounts/01-assert.yaml

```diff
@@ -1,3 +1,7 @@
+apiVersion: kuttl.dev/v1beta1
```

Comment:
@iam-veeramalla usual timeout for parallel tests is 1200s. By adding a timeout in this file, it'll be overridden. Is this intentional?

Reply:
I think, this test is failing due to mismatch in the order of the volume mounts.

Reply:
Yeah, the test is passing now. We can probably get rid of the timeout